int32 servoClockWiseRotation
int32 servoDegreeRotation

